src/stream.rs

```diff
@@ -8,7 +8,7 @@ use Compression;
 use ffi;
 
 pub struct Stream<D: Direction> {
-    raw: ffi::mz_stream,
+    pub raw: ffi::mz_stream,
```

Comment:
There's a `total_in` method below which perhaps can be used to avoid this `pub`?